simulation/modules/macrophage.py

```diff
@@ -3,83 +3,264 @@
 import attr
 import numpy as np
 
-from simulation.cell import CellData
-from simulation.coordinates import Point
+from simulation.cell import CellData, CellList
+from simulation.coordinates import Point, Voxel
 from simulation.grid import RectangularGrid
 from simulation.module import Module, ModuleState
-from simulation.modules.afumigatus import AfumigatusCellData, AfumigatusCellList
+from simulation.modules.fungus import FungusCellData, FungusCellList
 from simulation.modules.geometry import TissueTypes
-from simulation.modules.phagocyte import PhagocyteCellData, PhagocyteCellList
+from simulation.plot import plot_cells_num
 from simulation.state import State
 
+MAX_CONIDIA = 100
+
+
+class MacrophageCellData(CellData):
```

Comment:
There are a ton of changes to this module incorporating the new model biology compared to the previous one. I am fine with this overwriting, but another option would be to have 2 different macrophage modules where the old is one version and the new is another and the user can configure which to run.